test/state_test.js

```diff
@@ -218,23 +244,26 @@ describe('State', () => {
     assume(spotRequests[0]).has.property('state', secondState);
   });
 
-  it('should be able to do a spot request upsert', async () => {
-    let firstState = 'open';
-    let secondState = 'closed';
-    defaultSR.state = firstState;
+  it('should be able to report an AMI\'s usage', async () => {
 
-    await db.upsertSpotRequest(defaultSR);
-    let spotRequests = await db.listSpotRequests(); 
-    assume(spotRequests).has.length(1);
-    assume(spotRequests[0]).has.property('state', firstState);
-
-    defaultSR.state = secondState;
-    await db.upsertSpotRequest(defaultSR);
-    spotRequests = await db.listSpotRequests(); 
-    assume(spotRequests).has.length(1);
-    assume(spotRequests[0]).has.property('state', secondState);
+    await db.reportAmiUsage({region: defaultSR.region, id: defaultSR.id});
```

Comment:
Let's use the ImageId instead of a spot request Id.  This isn't important now (it's just a string) but when things go wrong in the future, it'll be confusing to see "sr-1234" when we're expecting "ami-1234".  The other thing is that we might start doing checks in the reportAmiUsage method to make sure that the imageId actually matches a pattern.  Not important, but it's also a pretty easy fix, using `defaultSR.imageId` instead of `.id`.